include/nuttx/mmcsd.h

```diff
@@ -95,6 +95,16 @@ struct mmc_ioc_cmd
   unsigned int blksz;
   unsigned int blocks;
 
+  /* Sleep at least postsleep_min_us useconds, and at most
+   * postsleep_max_us useconds *after* issuing command.  Needed for
+   * some read commands for which cards have no other way of indicating
+   * they're ready for the next command (i.e. there is no equivalent of
+   * a "busy" indicator for read operations).
+   */
+
+  unsigned int postsleep_min_us;
+  unsigned int postsleep_max_us;
```

Comment:
Since it is only necessary for some "limited" SDcards, isn't it better to wrap it around a config flag?

Reply:
This is used to align with Linux uapi definition for compatibility reason. This structure is normally defined on the stack,   so it isn't very useful to add a new flag to save the space.